README.md

```diff
@@ -1,24 +1,54 @@
 # Lazyman.bundle
 Lazyman Plex Channel to access Lazyman streams along with recaps and extended highlights.
 
-## Streaming and transcoding
+
+## Streaming and Transcoding
 As far as I gather, the Plex Media Server where the plug-in is installed need to be powerful enough to transcode the HLS (HTTP Live Streaming) streams.
 It might work with a transcoding client such as Plex Media Player or OpenPHT on a computer, not verified.
 Installing the plug-in on a weaker NAS and using a limited client will not work.
 
-### Recaps and highlights
-Recaps and highlights are not part of the Lazyman solution but is offered in the same NHL API and are added for convenience. These are MP4 files and not streams so they should generally work better on most devices. They will also work (or not work) without `/etc/hosts` changes.
+### Recaps and Highlights
+Recaps and highlights are not part of the Lazyman solution but is offered in the same NHL API and are added for convenience. These are MP4 files and not streams so they should generally work better on most devices. They will also work (or not work) without changes to your `hosts` file.
+
+
+## Prerequisites
+### Modify hosts file
+ * Modify your `hosts` file to work with Lazyman on the machine running Plex Media Server:
+ 1. From a command prompt or terminal window, type `ping nhl.freegamez.gq` and note the IP address supplied in the output.
```

Comment:
How official is this? Lazyman source seems to look up nhl.zipstreams.net ? Maybe list both?

Reply:
I didn't realize the source had shifted to the new domain. I gathered the URL from a r/LazyMan post that seemed to help a lot of people who were having hosts file issues. Happy to list both, but now I'm wondering if the entry I included might be going away at some point.
@StevensNJD4 Do you have any guidance on this?

Reply:
It looks like both URLs currently resolve to the same IP but I agree that I'd like to know if that's potentially going to change (or if one of them is going to go away).

Reply:
use nhl.zipstreams.net to get the ip

Reply:
Will do. Thanks for the input @StevensNJD4 .

Reply:
Domain adjusted.